src/tests/components/AddRecipe.test.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import { render, cleanup, fireEvent, waitFor } from '@testing-library/react';
+import '@testing-library/jest-dom';
+import axiosMock from 'axios';
+import AddRecipe from '../../components/AddRecipe';
+
+describe('AddRecipe', () => {
+  afterEach(cleanup);
+
+  it('renders', () => {
+    const { asFragment } = render(<AddRecipe />);
+    expect(asFragment()).toMatchSnapshot();
+  })
+
+  describe('has form for adding a recipe', () => {
+
+    it('should be a class for a submittable form containing name, description and ingredients',
+      () => {
+        const {
+          getByTestId,
+          getByLabelText,
+        } = render(<AddRecipe />);
+
+        expect(getByTestId('add-recipe-root')).toHaveClass('submit-form');
+        expect(getByLabelText('Name')).toBeVisible()
+        expect(getByLabelText('Description')).toBeVisible();
+        expect(getByLabelText('Ingredients (separate them with a comma)')).toBeVisible();
+      });
+
+    it('should submit successfully', async () => {
```

Comment:
The first of these two tests doesn't provide a lot of value by itself and would be better as part of the second one as an integration test. This is a style of testing we follow because the author of react-testing-library has put [this philosophy](https://kentcdodds.com/blog/write-tests) into creating the library so it's something we follow and has proven to be very robust